modules/couchdb/couchdb.go

```diff
@@ -65,6 +66,9 @@ func (cdb *CouchDB) Init() bool {
 		return false
 	}
 
+	urlPathOverviewStats = fmt.Sprintf(urlPathOverviewStats, cdb.Config.Node)
+	urlPathSystemStats = fmt.Sprintf(urlPathSystemStats, cdb.Config.Node)
```

Comment:
please move it to appropriate scape* method, do it there

Reply:
All global variables are shared across all instances of couchdb collector, not to a particular jobs (which is our case). Should be local to scrape* function.

Reply:
Fixed